frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/licenses/LicensesPortlet.java

```diff
@@ -317,6 +359,9 @@ private License updateLicenseFromRequest(License license, ActionRequest request)
         license.setGPLv2Compat(gpl2compatibility);
         license.setGPLv3Compat(gpl3compatibility);
         license.setChecked(checked);
+        String obligationIds = request.getParameter("obligations");
+        List<String> oblIds = obligationIds == null || obligationIds.equals("") ? Lists.newArrayList() : Arrays.asList(obligationIds.split(","));
```

Comment:
Is it possible to use `CommonUtils.isNotNullEmptyOrWhitespace()` same for line 457

Reply:
Have used